pkg/systemd/host.js

```diff
@@ -488,7 +489,7 @@ PageServer.prototype = {
             // if system is unregistered, always show that
             if (self.unregistered) {
                 self.os_updates_icon.className = "pficon pficon-warning-triangle-o";
-                set_page_link("#system_information_updates_text", "subscriptions", _("System Not Registered"));
+                set_page_link("#system_information_updates_text", "subscriptions", _("System Does Not Receive Updates"));
```

Comment:
A bit on the fence about this change, and while a valuable fix, not directly related to Insights.
The link will take you to the subscriptions page, and it doesn't mention anything about updates there, only registration.
I would change this only if we also change the subscription page, maybe also related to merging the software updates and subscription page.

Reply:
The same is true for "Not connected to Insights": The link will take you to the subscriptions page, and it doesn't mention anything about Insights there, only registration.

The idea was that we put a banner on the subscriptions that explains what this is all about.